app/scripts/modules/core/src/managed/UnpinArtifactModal.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { memo, useCallback } from 'react';
```

Comment:
you'll note this file looks... very similar to the `PinArtifactModal` component. I explicitly decided to duplicate instead of trying to re-use chunks because in my experience trying to go to reusable chunks too fast in these cases is often counter productive and leads to headaches and refactors. I also expect things to diverge even further between the two as we add more features to both flows. Either way we can always consolidate in the future if it turns out to be useful